doc/lsst.analysis.ap/tasks/lsst.analysis.ap.PlotImageSubtractionCutoutsTask.rst

```diff
@@ -100,6 +100,23 @@ until then, look at ``data/association-flag-map.yaml`` in `lsst.ap.association`
 
     Cutout with metadata annotations, and all catalog flags set.
 
+**Generating Multi-size Cutouts**
+---------------------------------
+
+Multi-size cutouts can be generated by setting the ``config.sizes`` key in config. The example image shown below contains cutouts of sizes 32x32 and 64x64 pixels.
```

Comment:
We generally use the "one sentence per line" for text wrapping:
https://developer.lsst.io/restructuredtext/style.html#text-wrapping

```suggestion
Multi-size cutouts can be generated by setting the ``config.sizes`` key.
The example image shown below contains cutouts of sizes 32x32 and 64x64 pixels.
```